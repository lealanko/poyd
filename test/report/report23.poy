(* Build 3 trees and report graphtrees to screen *)
read ("1.fas","30.ss" )
set(seed:50)
build (3)
report (graphtrees)
