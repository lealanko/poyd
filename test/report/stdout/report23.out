Tree with cost 2335
  |-----------------------------------------------------t1   
  |                                                          
  | |---------------------------------------------------t2   
  | |                                                        
  | |   |-----------------------------------------------t27  
  | |   |                                                    
  | | |-| |---------------------------------------------t25  
  | | | |-|                                                  
  | | |   |---------------------------------------------t26  
  | | |                                                      
  | | |   |---------------------------------------------t28  
  | | | |-|                                                  
  | | | | |---------------------------------------------t29  
  | | | |                                                    
  | | | |   |-------------------------------------------t40  
  | | | |   |                                                
  | | | |   |   |---------------------------------------t6   
  | | | |   |   |                                            
  | | | |   | |-| |-------------------------------------t18  
  | | | |   | | |-|                                          
  | | | | |-| |   |-------------------------------------t20  
  | | | | | | |                                              
  | | | | | |-| |---------------------------------------t11  
  | | | | |   | |                                            
  | | | | |   |-| |-------------------------------------t38  
  | | | | |     |-|                                          
  | | | | |       |-------------------------------------t47  
  | | | | |                                                  
  | | | | |   |-----------------------------------------t49  
  | | | | |   |                                              
  | | | | |   | |---------------------------------------t33  
  | | | | |   | |                                            
  | | | | |   | | |-------------------------------------t4   
  | | | | | |-| | |                                          
  | | | | | | |-| | |-----------------------------------t5   
  | | | | | |   |-| |                                        
  | | | | | |     |-| |---------------------------------t34  
  | | | | | |       |-|                                      
  | | | | | |         |---------------------------------t7   
  | | | | | |                                                
  | | | | | |     |-------------------------------------t3   
  | | | | | |   |-|                                          
  | | | | | |   | |-------------------------------------t32  
  | | | | | |   |                                            
  | | | | | | |-| |-------------------------------------t9   
  | | | | | | | | |                                          
  | | | | | | | |-| |-----------------------------------t36  
  | | | | | | |   |-|                                        
--| | | | | | |     |-----------------------------------t8   
  |-| | | | | |                                              
    | | | | | |     |-----------------------------------t16  
    |-| | | | |     |                                        
      | | | | |     | |---------------------------------t15  
      |-| | | |   |-| |                                      
        | | | |   | |-| |-------------------------------t35  
        | | | |   |   |-|                                    
        | | | |   |     |-------------------------------t45  
        | | | |   |                                          
        |-| | |   | |-----------------------------------t37  
          | | | |-| |                                        
          | | | | | |   |-------------------------------t12  
          | | | | | | |-|                                    
          | | | | |-| | |-------------------------------t14  
          | | | |   |-|                                      
          |-| | |     | |-------------------------------t13  
            | | |     |-|                                    
            | | |       |-------------------------------t48  
            | | |                                            
            | | | |-------------------------------------t46  
            | | | |                                          
            |-| | | |-----------------------------------t39  
              | | | |                                        
              | | | | |---------------------------------t10  
              | | | | |                                      
              | | | | | |-------------------------------t17  
              | | | | | |                                    
              | | | | | | |-----------------------------t42  
              |-| | | | | |                                  
                | | | | | | |---------------------------t31  
                | | | | | | |                                
                | | | | | | |   |-----------------------t23  
                | | | | | | |   |                            
                |-| | | | | | |-| |---------------------t21  
                  |-| | | | | | |-|                          
                    |-| | | | |   |---------------------t24  
                      |-| | | |                              
                        |-| | | |-----------------------t22  
                          |-| | |                            
                            | | |   |-------------------t43  
                            |-| | |-|                        
                              | | | |-------------------t44  
                              |-| |                          
                                |-| |-------------------t41  
                                  | |                        
                                  |-| |-----------------t19  
                                    |-|                      
                                      |-----------------t30  
Tree with cost 2318
  |-----------------------------------------t1   
  |                                              
  | |---------------------------------------t2   
  | |                                            
  | |   |-----------------------------------t28  
  | | |-|                                        
  | | | |-----------------------------------t29  
  | | |                                          
  | | | |-----------------------------------t37  
  | | | |                                        
  | | | |   |-------------------------------t15  
  | | | |   |                                    
  | | | |   | |-----------------------------t16  
  | | | | |-| |                                  
  | | | | | |-| |---------------------------t35  
  | | | | |   |-|                                
  | | | | |     |---------------------------t45  
  | | | | |                                      
  | | | | |   |-----------------------------t19  
  | | | | |   |                                  
  | | | | |   | |---------------------------t41  
  | | | | |   | |                                
  | | | | |   | | |-------------------------t30  
  | | | | |   | | |                              
  | | | | |   | | |   |---------------------t18  
  | | | | |   | | | |-|                          
  | | | | |   | | | | |---------------------t20  
  | | | | |   | | | |                            
  | | | | |   | | | |   |-------------------t27  
  | | | | |   | | | |   |                        
  | | | | |   | | | | |-| |-----------------t25  
  | | | | | |-| | | | | |-|                      
  | | | | | | |-| | | |   |-----------------t26  
  | | | | | |   |-| | |                          
  | | | | | |     |-| | |-------------------t33  
  | | | | | |       | | |                        
  | | | | | |       | | | |-----------------t40  
  | | | | | |       |-| | |                      
  | | | | | |         | | |   |-------------t11  
  | | | | | |         |-| | |-|                  
  | | | | | |           |-| | |-------------t6   
  | | | | | |             |-|                    
  | | | | | |               | |-------------t38  
  | | | | | |               |-|                  
  | | | | | |                 |-------------t47  
  | | | | | |                                    
  | | | | | |     |-------------------------t21  
  | | | | | |   |-|                              
--| | | | | |   | |-------------------------t24  
  |-| | | | |   |                                
    |-| | | |   |   |-----------------------t10  
      | | | |   | |-|                            
      |-| | |   | | |-----------------------t39  
        | | |   |-|                              
        |-| |   | | |-----------------------t17  
          | |   | |-|                            
          | | |-|   |-----------------------t42  
          | | | |                                
          | | | | |-------------------------t23  
          | | | | |                              
          | | | | | |-----------------------t22  
          | | | | | |                            
          | | | |-| | |---------------------t31  
          |-| |   |-| |                          
            | |     |-| |-------------------t43  
            | |       |-|                        
            | |         |-------------------t44  
            | |                                  
            | |   |-------------------------t36  
            | |   |                              
            | | |-| |-----------------------t8   
            | | | |-|                            
            | | |   |-----------------------t9   
            | | |                                
            | | |   |-----------------------t3   
            | | | |-|                            
            |-| | | |-----------------------t32  
              | | |                              
              | | |     |-------------------t12  
              | | |   |-|                        
              | | |   | |-------------------t14  
              | | | |-|                          
              | | | | | |-------------------t13  
              |-| | | |-|                        
                | | |   |-------------------t48  
                |-| |                            
                  | |   |-------------------t46  
                  | | |-|                        
                  |-| | |-------------------t49  
                    | |                          
                    | | |-------------------t4   
                    |-| |                        
                      | | |-----------------t5   
                      |-| |                      
                        |-| |---------------t34  
                          |-|                    
                            |---------------t7   
Tree with cost 2325
  |--------------------------------------t1   
  |                                           
  | |------------------------------------t2   
  | |                                         
  | |   |--------------------------------t29  
  | |   |                                     
  | |   |   |----------------------------t27  
  | |   |   |                                 
  | |   | |-| |--------------------------t25  
  | |   | | |-|                               
  | |   | |   |--------------------------t26  
  | |   | |                                   
  | |   | | |----------------------------t28  
  | |   | | |                                 
  | |   | | |   |------------------------t33  
  | | |-| | |   |                             
  | | | | | | |-| |----------------------t49  
  | | | |-| | | |-|                           
  | | |   | | |   |----------------------t6   
  | | |   |-| |                               
  | | |     | | |------------------------t40  
  | | |     |-| |                             
  | | |       | | |----------------------t11  
  | | |       |-| |                           
  | | |         |-| |--------------------t38  
  | | |           |-|                         
  | | |             |--------------------t47  
  | | |                                       
  | | |   |------------------------------t15  
  | | |   |                                   
  | | |   | |----------------------------t45  
  | | | |-| |                                 
  | | | | |-| |--------------------------t16  
  | | | |   |-|                               
  | | | |     |--------------------------t35  
  | | | |                                     
  | | | |   |----------------------------t37  
  | | | |   |                                 
  | | | |   |   |------------------------t12  
  | | | |   | |-|                             
  | | | | |-| | |------------------------t14  
  | | | | | |-|                               
  | | | | |   | |------------------------t13  
  | | | | |   |-|                             
  | | | | |     |------------------------t48  
  | | | | |                                   
  | | | | |     |------------------------t31  
  | | | | |     |                             
--| | | | |     |   |--------------------t10  
  |-| | | |     | |-|                         
    | | | |   |-| | |--------------------t39  
    | | | |   | |-|                           
    | | | |   |   | |--------------------t17  
    | | | |   |   |-|                         
    | | | |   |     |--------------------t42  
    |-| | |   |                               
      | | |   | |------------------------t3   
      | | |   | |                             
      | | | |-| |   |--------------------t4   
      | | | | | | |-|                         
      | | | | | | | |--------------------t46  
      | | | | |-| |                           
      | | | |   |-| |--------------------t5   
      |-| | |     | |                         
        | | |     |-| |------------------t34  
        | | |       |-|                       
        | | |         |------------------t7   
        | | |                                 
        |-| |   |------------------------t23  
          | |   |                             
          | | |-| |----------------------t21  
          | | | |-|                           
          | | |   |----------------------t24  
          | | |                               
          | | | |------------------------t22  
          | | | |                             
          | | | |   |--------------------t43  
          |-| | | |-|                         
            | | | | |--------------------t44  
            | | | |                           
            | | | |   |------------------t19  
            | | | |   |                       
            | | | |   | |----------------t30  
            |-| | | |-| |                     
              | | | | |-| |--------------t18  
              |-| | |   |-|                   
                |-| |     |--------------t20  
                  | |                         
                  | | |------------------t41  
                  |-| |                       
                    | | |----------------t32  
                    | | |                     
                    |-| | |--------------t9   
                      |-| |                   
                        |-| |------------t36  
                          |-|                 
                            |------------t8   
